.env

```diff
@@ -74,3 +74,7 @@ ROUTEMASTER_MONITOR_INTERVAL=10
 ROUTEMASTER_SCALING_THRESHOLD=100
 ROUTEMASTER_SCALING_DEADLINE=1000
 
+# maximum number of bytes in an event's optional data payload (when messagepack'ed)
+# DO NOT INCREASE THIS - using the event bus as a data bus is a strong smell of
+# an incompatible architecture, and will cause major bus scalability issues.
+ROUTEMASTER_MAX_EVENT_DATA=64
```

Comment:
@tompave — thanks. To @gregbeech's point, I have to say I agree. If we compromise further we'll break down semantics of the architecture. The payload should really be identical to the blob the API would serialize, so we'll have to make the default bigger.

Unfortunately that means our data blob limit has to be at least 144 bytes (from Greg's example), probably best rounded to 256 bytes.

Makes me uncomfortable, but perhaps if we add monitoring of event sizes?